qhub/template/{{ cookiecutter.repo_directory }}/infrastructure/kubernetes.tf

```diff
@@ -235,6 +235,12 @@ module "prefect" {
   {% if cookiecutter.prefect.image is defined -%}
   image                = "{{ cookiecutter.prefect.image }}"
   {% endif -%}
+  {% if cookiecutter.prefect.agent.prefectLabels is defined -%}
+  image                = "{{ cookiecutter.prefect.agent.prefectLabels }}"
```

Comment:
Think this will need to be changed to 

```
agent = {
{% if cookiecutter.prefect.agent.prefectLabels is defined -%}
     prefectLabels = "{{ cookiecutter.prefect.agent.prefectLabels }}"
{% endif %}
...
}
```